website/docs/advanced/targeting/percentage.md

```diff
@@ -0,0 +1,122 @@
+---
+id: percentage
+title: Percentage-based Targeting
+description: With percentage-based user targeting is a technique used in software development and feature management to gradually release a new feature to a subset of users.
+---
+
+Percentage-based targeting, also known as percentage rollouts or feature rollouts, is a technique used in software development and feature management to gradually release a new feature to a subset of users. Instead of releasing a feature to all users at once, a specific percentage of users are selected to receive access to the feature. This allows developers to control and monitor the impact of the new feature in a controlled manner.
+
+### Enable feature
+
+1. <a href="https://app.configcat.com/auth/login" target="_blank">Log in</a> to access the _Dashboard_
+2. Go to **Feature Flags & Settings**
+3. Select **TARGET % OF USERS** after clicking the actions icon.
+
+<img src="/docs/assets/targeting-2.png" className="zoomable" alt="targeting-2" />
+
+## Anatomy of the percentage-based targeting
+
+Percentage based targeting consists of **% value** and the **Served value** pairs.
+
+### % value
+
+Any _number between 0 and 100_ that represents a randomly allocated fraction of your users.
+
+### Served value
+
+The exact value that will be served to the users that fall into that fraction. Depending on the kind of your setting this could be:
+
+| Setting Kind   | Setting Type | Description                                     |
+| -------------- | ------------ | ----------------------------------------------- |
+| On/Off Toggle  | Boolean      | true/false, usually the state of a feature flag |
+| Text           | String       | any `string`, max. 100000 characters             |
+| Whole Number   | Integer      | any whole number within the range of `Int32`    |
+| Decimal Number | Double       | any decimal number within the range of `double` |
+
+## Stickiness & Consistency
+
+The percentage-based targeting is sticky by design and consistent across all SDKs.
+
+Percentage-based targeting is based on the identifier of the `User Object` passed to the SDK's `getValue()` methods.
```

Comment:
Mostmár el lehet téríteni a percentage evaluation attribute-ot, lehet érdemes megjegyezni itt is.